### ISSUE TITLE
Hi,guys,How can I use pytorch as backend-only? 

### ISSUE THREAD

**Issue by heartInsert** [keras-team/autokeras#625]

**Comment by JaeDukSeo** [comment]
I think you can set the constant value

# Shuffle the data and set the settings
from sklearn.utils import shuffle
x_train,y_train = shuffle(x_train,y_train)

from autokeras.constant import Constant
# Constant.BACKEND = 'tensorflow'
Constant.MAX_BATCH_SIZE = 6
Constant.VALIDATION_SET_SIZE = 0.3

from autokeras.image.image_supervised import ImageClassifier

something like that

**Comment by heartInsert** [comment]
> I think you can set the constant value
> 
> # Shuffle the data and set the settings
> from sklearn.utils import shuffle
> x_train,y_train = shuffle(x_train,y_train)
> 
> from autokeras.constant import Constant
> 
> # Constant.BACKEND = 'tensorflow'
> Constant.MAX_BATCH_SIZE = 6
> Constant.VALIDATION_SET_SIZE = 0.3
> 
> from autokeras.image.image_supervised import ImageClassifier
> 
> something like that

Thanks  for you reply  ,  I  have  solved  it
